CITATION.cff

```diff
@@ -19,6 +19,10 @@ license: BSD-3-Clause
 contact:
 - email: GAMMAPY-COORDINATION-L@IN2P3.FR
   name: "Coordination committee of the Gammapy project"
+identifiers:
+- description: "Zenodo repository of the release 1.1"
```

Comment:
Should be 1.2 no?

Reply:
When it will be tagged, yes, of course... Should I do it right now, @registerrier ?